cmd/mattermost/commands/command.go

```diff
@@ -67,3 +76,33 @@ func moveCommandCmdF(command *cobra.Command, args []string) error {
 func moveCommand(a *app.App, team *model.Team, command *model.Command) *model.AppError {
 	return a.MoveCommand(team, command)
 }
+
+func listCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("Enter at least one team")
+	}
+
+	teams := getTeamsFromTeamArgs(a, args)
```

Comment:
I think would be nice to allow to list from all teams, like here: https://github.com/mattermost/mattermost-server/pull/9528/files#diff-a2d1b2bd4062c22fea5fdc1e88b50fe4R31

Reply:
@jespino 
I will address that.